plugins/palo_alto_cortex_xdr/icon_palo_alto_cortex_xdr/tasks/monitor_alerts/task.py

```diff
@@ -125,6 +125,13 @@ def get_alerts_palo_alto(self, state: dict, start_time: int, end_time: int, aler
             state[QUERY_START_TIME] = query_start_time
             state[QUERY_END_TIME] = query_end_time
```

Comment:
Can we remove this line completely now?

Reply:
Yes I think so, I'll test it again to be sure